### PR TITLE
fix: sattelite floor for extended genesis parcels

### LINKED ISSUE #358: Missing default floor on scenes
### **Issue Description:**
Some scenes do no have a geometry floor and they rely on the previous green floor shader to act as a base ground. Right now, we dont have that and we display the satellite view as ground. We need to mask the satellite view with ground texture for all the scenes around the player that are loading any type of LOD, or at least LOD0 and Scene Runtime

### **Steps to reproduce:**

  1. Go to -6,-15
  2. Check the floor

### **Reproduction rate:**
- [ ] < 10%
- [ ]  < = 25%
- [ ]  50%
- [ ]  < = 75%
- [ x]  100%

### **Severity:**

- [ x] Cosmetic
- [ ] Minor
- [ ] Medium
- [ ] High
- [ ] Critical
- [ ] Blocker

### **Priority:**
- [ ] High
- [x ] Medium
- [ ] Low

#### **Evidence:**
image
image

### LINKED ISSUE #745: [QA] Cliff colliders prevent the camera to look upfront and place the hands.
Approaching the limits of the world as colliders have collateral consecuences, as the player cannot look at himself frontally, and also the avatar places the hands on an invisible wall.

Could this be calculated procedurally insteado of via collider?
Could the collider be flagged as invisible so it doesnt affect the camera or the IK?

image

### LINKED ISSUE #1025: [QA] North-east corner extra parcels do not have floor
The parcels in the north-east part that extends beyond the main square of genesis city is are not being covered by the satellite floor, showing a gap with the ocean while loading the scenes.

image
image

Some of the scene never loads due missing LOD, leaving a big gap there

image

### PR DESCRIPTION
**Pull request by popuz**
## What does this PR change?
fix #358
fix #745
fix #1025

I moved base Terrain just a bit below Infinite/SatteliteFloor and removed making holes on occupied parcels. Such terrain only generate texture for it, so it blends better than InfiniteFloor. For long distance, floor is still better, because of the sattelite pictures.

image

## How to test the changes?

1. See mentioned issues - they have repro steps.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

File: Explorer/Assets/DCL/Landscape/TerrainFactory.cs [hunk 1]
```diff
@@ -72,6 +72,7 @@ public Collider CreateBorderCollider(string name, Transform parent, Vector3 size
         {
             BoxCollider collider = new GameObject(name).AddComponent<BoxCollider>();
             collider.transform.SetParent(parent);
+            collider.gameObject.layer = LayerMask.NameToLayer("InvisibleColliders");
```
Comment (davidejensen) [inline Explorer/Assets/DCL/Landscape/TerrainFactory.cs:75]: Maybe we can create a class with some constants like we are doing with the hashes of animators

**Review by davidejensen — APPROVED** [review]
Approved with 1 minor suggestion :)

**Review by LucasLioyQA — APPROVED** [review]
**QA checked*

358: 
vitaly one part two

Even though it is the same as in main currently, it was said to be expected

745:

https://github.com/decentraland/unity-explorer/assets/146749313/78d661a8-ca06-42bb-8313-e571b59aab78

1025:

vitaly three pic

File: Explorer/Assets/DCL/Landscape/TerrainFactory.cs (hunk 1 above)
Comment (popuz) [reply Explorer/Assets/DCL/Landscape/TerrainFactory.cs:75]: would be nice, but maybe in more calm times.

**Pull request merged**